services/galley/src/Galley/Data/Queries.hs

```diff
@@ -239,8 +255,8 @@ selectSrv = "select base_url, auth_token, fingerprints, enabled from service whe
 
 -- Bots ---------------------------------------------------------------------
 
-insertBot :: PrepQuery W (ConvId, BotId, ServiceId, ProviderId) ()
-insertBot = "insert into member (conv, user, service, provider, status) values (?, ?, ?, ?, 0)"
+insertBot :: PrepQuery W (ConvId, BotId, ServiceId, ProviderId, RoleName) ()
+insertBot = "insert into member (conv, user, service, provider, status, conversation_role) values (?, ?, ?, ?, 0, ?)"
```

Comment:
Please document how/why bots need a conversation_role. Don't they already not have access to that part of the API? At the moment they're whitelisted to only make requests to `/bot/` endpoints. Maybe revert this change?

Reply:
Agreed, removed references to any roles when bots are concerned.